localstack/services/providers.py

```diff
@@ -258,7 +258,7 @@ def route53resolver():
     )
 
 
-@aws_provider()
+@aws_provider(api="s3", name="default")
```

Comment:
The params for the decorator here actually aren't necessary, since the default for `api` is the function's name (`s3`) and the default for `name` is `default`.

Reply:
Thanks! It makes a lot of sense. I will update once the tests run, there should not be any issue but I'd rather be certain 😅